Makefile-docker

```diff
@@ -77,6 +77,25 @@ update_assets:
 	$(PYTHON_COMMAND) manage.py collectstatic --noinput
 
 
+# Build list of dependencies to install
+DEPS = pip prod
+# If we're running a development image, then we should install the development dependencies
+ifeq ($(DOCKER_TARGET), development)
+DEPS += dev
+endif
```

Comment:
Should these lines be at the top of the file with other file-level constants, or inside `update_deps`?  (they're only referenced in `update_deps`

Reply:
Good point. I moved them up and also that means we can use them for the check command also.